spec/queries/popular_onet_codes_query_spec.rb

```diff
@@ -2,11 +2,14 @@
 
 RSpec.describe PopularOnetCodesQuery do
   it "returns the onet codes for popular occupation standards" do
-    create(:occupation_standard, title: "Medical Assistant", onet_code: "11")
-    create_list(:occupation_standard, 2, title: "Pipe Fitter", onet_code: "22")
-    create_list(:occupation_standard, 3, title: "Nurse", onet_code: "33")
+    create(:occupation_standard, :with_work_processes, title: "Medical Assistant", onet_code: "11")
+    create(:occupation_standard, title: "HR Specialist", onet_code: "12")
+    create_list(:occupation_standard, 2, :with_work_processes, title: "Pipe Fitter", onet_code: "22")
+    create_list(:occupation_standard, 3, :with_work_processes, title: "Nurse", onet_code: "33")
     create(:occupation_standard, onet_code: nil)
 
+    stub_const("PopularOnetCodesQuery::LIMIT", 4)
+
```

Comment:
For line 13 below, can we run this without passing `limit: 3`, so it runs with the default limit of 4, therefore testing that the occupation standard with onet code 12 is not getting returned because it doesn't have a work process, not because it was 4th to be returned when our limit was only 3?